### ISSUE TITLE
Packager gets stuck unless all k8s yamls are in a folder called "templates"

### ISSUE THREAD

**Issue by borg286** [abrisco/rules_helm#106]
If you have your k8s yaml files not in a folder called "templates" then the step where the packager packages up the helm chart then it simply hangs forever taking 100% cpu, likely in a loop.
In the bazel-way you can have your files in whatever structure you want, then simply point to them when invoking a bazel rule. But there still seems to be an implicit reliance that all templates are in a folder called "templates". Perhaps that is a fundamental problem with the helm binary, so you might have to work around that limitation.

**Comment by purkhusid** [comment]
Yes, I think it makes sense to change the behavior so that instead of expecting the template files to already be in the `templates` folder that they would just be moved into the templates folder which the package binary creates. 

An example is where you have a generated file you want to add to the `templates`.
